spark-jobs/src/main/scala/za/co/absa/enceladus/conformance/ConformanceExecution.scala

```diff
@@ -162,7 +156,7 @@ trait ConformanceExecution extends CommonJobExecution {
 
     val minBlockSize = configReader.getLongOption(CommonConfConstants.minPartitionSizeKey)
     val maxBlockSize = configReader.getLongOption(CommonConfConstants.maxPartitionSizeKey)
-    val withRepartitioning = repartitionDataFrame(result, minBlockSize, maxBlockSize)
+    val withRepartitioning = repartitionDataFrame(withPartCols, minBlockSize, maxBlockSize)
```

Comment:
Is my understanding of the PR correct, that this is the problematic place where, originally, the dataframe without `Info*` columns was used?

(Otherwise just extracting to a method etc. is just readablity stuff)